ivre/db/mongo.py

```diff
@@ -1164,6 +1210,19 @@ def searchcategory(cat, neg=False):
         return {'categories': cat}
 
     @staticmethod
+    def searchlabel(group, label=None, neg=False):
```

Comment:
It would be nice if both `group` and `label` could be regexp.

Also does `neg` work both when there is a matching `group` without a matching `label` and when there is no matching `group`?